Thu Dec 7 12:50:13 IST 2023

SLURM_JOBID:		 999051
SLURM_JOB_NODELIST:	 cs-3090-01 


python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 10 --strains 4  --dense 4 --CMDprogress_0_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 14 --strains 4  --dense 8 --CMDprogress_1_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 10 --strains 4  --identicalf 0.01 --CMDprogress_2_239
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 180, in <module>
    utils.log_print(printFlag,"\n\n\n.........................initialize..........................")
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: '\n\n\n.........................initialize..........................'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 181, in <module>
    utils.log_print(printFlag,"FileName:", fileName)
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: 'FileName: AA/AA_BA_identical_14_4_0_8_50_10.0_140000'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 182, in <module>
    utils.log_print(printFlag,"Strains:", paras.strains)
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: 'Strains: 4'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 183, in <module>
    utils.log_print(printFlag,"Using", paras.randomGraph,"random graph model")
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: 'Using BA random graph model'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 184, in <module>
    utils.log_print(printFlag,"Num of links:", G.number_of_edges())
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: 'Num of links: 336'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 185, in <module>
    utils.log_print(printFlag,"Reverse ZERO:", torch.sum(torch.abs(Zmat2-Aw)))
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "Reverse ZERO: tensor(4.7311e-07, device='cuda:0')"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 186, in <module>
    utils.log_print(printFlag,"Num links weight bigger than 0.1:", torch.sum(Zmat>0.1)-paras.n)
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "Num links weight bigger than 0.1: tensor(0, device='cuda:0')"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 187, in <module>
    utils.log_print(printFlag,"Num links weight bigger than 0.05:", torch.sum(Zmat>0.05)-paras.n)
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "Num links weight bigger than 0.05: tensor(0, device='cuda:0')"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 188, in <module>
    utils.log_print(printFlag,"Num links weight bigger than 0.01:", torch.sum(Zmat>0.01)-paras.n)
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "Num links weight bigger than 0.01: tensor(345, device='cuda:0')"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 189, in <module>
    utils.log_print(printFlag,".............................................................")
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: '.............................................................'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 190, in <module>
    utils.log_print(printFlag,losses[-1]/timeHorizon*100)#
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: '1.2637992767849939e-05'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 204, in <module>
    utils.log_print(printFlag,paras.taus[0: paras.strains])
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: '[7.542090252618508, 7.400696194766855, 7.570360426163584, 7.418720572455853]'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 205, in <module>
    utils.log_print(printFlag,paras.R0s[0: paras.strains])
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: '[8.295010971555538, 8.561121290806776, 8.37067588893102, 7.877412935248897]'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 206, in <module>
    utils.log_print(printFlag,myEpi.taus[0])
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "tensor([7.5427, 7.4022, 7.5893, 7.4189], device='cuda:0',\n       grad_fn=<SelectBackward0>)"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 207, in <module>
    utils.log_print(printFlag,(myEpi.taus*myEpi.R0dTaus)[0])
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "tensor([8.2920, 8.5581, 8.3792, 7.8743], device='cuda:0',\n       grad_fn=<SelectBackward0>)"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 213, in <module>
    utils.log_print(printFlag,"err1:", torch.sqrt((PreA-Aw)**2).sum())
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "err1: tensor(6.0493, device='cuda:0', grad_fn=<SumBackward0>)"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 214, in <module>
    utils.log_print(printFlag,"err2:", torch.abs(PreAse-Awse).sum())
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "err2: tensor(428., device='cuda:0')"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 215, in <module>
    utils.log_print(printFlag,"cosine similarity:", evaluate.cosine_similarity(Aw, PreA))
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "cosine similarity: tensor(0.7544, device='cuda:0', grad_fn=<DivBackward0>)"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 216, in <module>
    utils.log_print(printFlag,"cosine similarity 2:", evaluate.cosine_similarity(Awse, PreAse))
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "cosine similarity 2: tensor(0.7036, device='cuda:0')"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 217, in <module>
    utils.log_print(printFlag,"spectral_analysis:", evaluate.spectral_analysis(Aw, PreA))
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "spectral_analysis: tensor(0.5805, device='cuda:0', grad_fn=<DivBackward0>)"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 219, in <module>
    utils.log_print(printFlag,"edge_correctness:", evaluate.edge_correctness(Awse, PreAse))
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "edge_correctness: tensor(0.7036, device='cuda:0')"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 220, in <module>
    utils.log_print(printFlag,"jaccard_index:", evaluate.jaccard_index(Awse, PreAse))
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "jaccard_index: tensor(0.5427, device='cuda:0')"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 221, in <module>
    utils.log_print(printFlag,torch.var(myEpi.taus, dim= 0))
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "tensor([1.5010e-10, 5.7978e-10, 3.7996e-10, 1.6569e-10], device='cuda:0',\n       grad_fn=<VarBackward0>)"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 222, in <module>
    utils.log_print(printFlag,torch.var(myEpi.R0dTaus, dim= 0))
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "tensor([4.8559e-08, 1.8681e-07, 1.8174e-07, 4.6437e-08], device='cuda:0',\n       grad_fn=<VarBackward0>)"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 223, in <module>
    utils.log_print(printFlag,torch.sum(predSignal[29, :, 0:-1])/paras.strains)
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "tensor(0.9998, device='cuda:0', grad_fn=<DivBackward0>)"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 224, in <module>
    utils.log_print(printFlag,torch.sum(signal[29, :, 1:])/paras.strains)
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "tensor(0.9998, device='cuda:0')"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 227, in <module>
    utils.log_print(printFlag,PreA[startV:startV+deltaV, startV:startV+deltaV])
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "tensor([[0.0000, 0.0029, 0.0027, 0.0042, 0.0055, 0.0087, 0.0048],\n        [0.0043, 0.0000, 0.0031, 0.0071, 0.0058, 0.0113, 0.0029],\n        [0.0011, 0.0003, 0.0000, 0.0027, 0.0045, 0.0046, 0.0010],\n        [0.0041, 0.0049, 0.0014, 0.0000, 0.0083, 0.0087, 0.0052],\n        [0.0095, 0.0086, 0.0085, 0.0128, 0.0000, 0.0177, 0.0098],\n        [0.0053, 0.0091, 0.0050, 0.0073, 0.0093, 0.0000, 0.0079],\n        [0.0072, 0.0037, 0.0061, 0.0088, 0.0111, 0.0135, 0.0000]],\n       device='cuda:0', grad_fn=<SliceBackward0>)"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 228, in <module>
    utils.log_print(printFlag,Aw[startV:startV+deltaV, startV:startV+deltaV])
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "tensor([[0.0000, 0.0000, 0.0000, 0.0000, 0.0100, 0.0100, 0.0100],\n        [0.0000, 0.0000, 0.0000, 0.0000, 0.0100, 0.0100, 0.0100],\n        [0.0000, 0.0000, 0.0000, 0.0000, 0.0100, 0.0100, 0.0000],\n        [0.0000, 0.0000, 0.0000, 0.0000, 0.0100, 0.0100, 0.0100],\n        [0.0100, 0.0100, 0.0100, 0.0100, 0.0000, 0.0100, 0.0100],\n        [0.0100, 0.0100, 0.0100, 0.0100, 0.0100, 0.0000, 0.0100],\n        [0.0100, 0.0100, 0.0000, 0.0100, 0.0100, 0.0100, 0.0000]],\n       device='cuda:0')"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 180, in <module>
    utils.log_print(printFlag,"\n\n\n.........................initialize..........................")
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: '\n\n\n.........................initialize..........................'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 181, in <module>
    utils.log_print(printFlag,"FileName:", fileName)
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: 'FileName: AA/AA_GEO_identical_10_4_0_4_50_10.0_140000'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 182, in <module>
    utils.log_print(printFlag,"Strains:", paras.strains)
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: 'Strains: 4'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 183, in <module>
    utils.log_print(printFlag,"Using", paras.randomGraph,"random graph model")
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: 'Using GEO random graph model'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 184, in <module>
    utils.log_print(printFlag,"Num of links:", G.number_of_edges())
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: 'Num of links: 200'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 185, in <module>
    utils.log_print(printFlag,"Reverse ZERO:", torch.sum(torch.abs(Zmat2-Aw)))
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "Reverse ZERO: tensor(2.7753e-07, device='cuda:0')"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 186, in <module>
    utils.log_print(printFlag,"Num links weight bigger than 0.1:", torch.sum(Zmat>0.1)-paras.n)
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "Num links weight bigger than 0.1: tensor(0, device='cuda:0')"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 187, in <module>
    utils.log_print(printFlag,"Num links weight bigger than 0.05:", torch.sum(Zmat>0.05)-paras.n)
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "Num links weight bigger than 0.05: tensor(0, device='cuda:0')"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 188, in <module>
    utils.log_print(printFlag,"Num links weight bigger than 0.01:", torch.sum(Zmat>0.01)-paras.n)
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "Num links weight bigger than 0.01: tensor(183, device='cuda:0')"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 189, in <module>
    utils.log_print(printFlag,".............................................................")
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: '.............................................................'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 190, in <module>
    utils.log_print(printFlag,losses[-1]/timeHorizon*100)#
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: '9.825409203282712e-06'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 204, in <module>
    utils.log_print(printFlag,paras.taus[0: paras.strains])
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: '[7.577506790998881, 7.548740489410478, 7.50449092197787, 7.532814664756424]'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 205, in <module>
    utils.log_print(printFlag,paras.R0s[0: paras.strains])
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: '[8.799698037717521, 8.263759972625767, 8.495236793364116, 8.22348679488616]'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 206, in <module>
    utils.log_print(printFlag,myEpi.taus[0])
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "tensor([7.5731, 7.5492, 7.5006, 7.5292], device='cuda:0',\n       grad_fn=<SelectBackward0>)"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 207, in <module>
    utils.log_print(printFlag,(myEpi.taus*myEpi.R0dTaus)[0])
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "tensor([8.7987, 8.2624, 8.4844, 8.2201], device='cuda:0',\n       grad_fn=<SelectBackward0>)"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 213, in <module>
    utils.log_print(printFlag,"err1:", torch.sqrt((PreA-Aw)**2).sum())
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "err1: tensor(0.8687, device='cuda:0', grad_fn=<SumBackward0>)"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 214, in <module>
    utils.log_print(printFlag,"err2:", torch.abs(PreAse-Awse).sum())
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "err2: tensor(20., device='cuda:0')"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 215, in <module>
    utils.log_print(printFlag,"cosine similarity:", evaluate.cosine_similarity(Aw, PreA))
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "cosine similarity: tensor(0.9679, device='cuda:0', grad_fn=<DivBackward0>)"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 216, in <module>
    utils.log_print(printFlag,"cosine similarity 2:", evaluate.cosine_similarity(Awse, PreAse))
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "cosine similarity 2: tensor(0.9778, device='cuda:0')"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 217, in <module>
    utils.log_print(printFlag,"spectral_analysis:", evaluate.spectral_analysis(Aw, PreA))
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "spectral_analysis: tensor(0.7875, device='cuda:0', grad_fn=<DivBackward0>)"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 219, in <module>
    utils.log_print(printFlag,"edge_correctness:", evaluate.edge_correctness(Awse, PreAse))
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "edge_correctness: tensor(0.9778, device='cuda:0')"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 220, in <module>
    utils.log_print(printFlag,"jaccard_index:", evaluate.jaccard_index(Awse, PreAse))
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "jaccard_index: tensor(0.9565, device='cuda:0')"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 221, in <module>
    utils.log_print(printFlag,torch.var(myEpi.taus, dim= 0))
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "tensor([1.8104e-10, 2.3180e-10, 1.3499e-10, 8.1875e-11], device='cuda:0',\n       grad_fn=<VarBackward0>)"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 222, in <module>
    utils.log_print(printFlag,torch.var(myEpi.R0dTaus, dim= 0))
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "tensor([9.5916e-08, 1.0255e-07, 8.8528e-08, 9.1884e-08], device='cuda:0',\n       grad_fn=<VarBackward0>)"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 223, in <module>
    utils.log_print(printFlag,torch.sum(predSignal[29, :, 0:-1])/paras.strains)
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "tensor(0.9924, device='cuda:0', grad_fn=<DivBackward0>)"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 224, in <module>
    utils.log_print(printFlag,torch.sum(signal[29, :, 1:])/paras.strains)
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "tensor(0.9924, device='cuda:0')"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 227, in <module>
    utils.log_print(printFlag,PreA[startV:startV+deltaV, startV:startV+deltaV])
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "tensor([[0.0000e+00, 4.0371e-04, 3.6746e-03, 1.2771e-02, 6.7626e-06, 1.1670e-06,\n         9.1267e-03],\n        [9.1407e-06, 0.0000e+00, 2.2295e-05, 1.5096e-06, 5.4342e-04, 8.7006e-07,\n         1.0417e-02],\n        [1.1837e-03, 1.3488e-03, 0.0000e+00, 7.9701e-04, 2.0323e-06, 6.0258e-07,\n         1.8260e-04],\n        [1.1262e-02, 4.9709e-05, 2.3029e-04, 0.0000e+00, 2.0050e-05, 1.8154e-06,\n         2.2992e-04],\n        [7.9434e-07, 5.1843e-07, 5.1221e-07, 1.8362e-06, 0.0000e+00, 6.6315e-06,\n         2.5145e-07],\n        [5.3545e-05, 1.9756e-06, 2.7138e-06, 1.7888e-05, 2.0795e-04, 0.0000e+00,\n         7.0346e-07],\n        [9.8489e-03, 9.1895e-03, 2.8565e-04, 1.1400e-05, 2.8529e-05, 1.0829e-06,\n         0.0000e+00]], device='cuda:0', grad_fn=<SliceBackward0>)"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 228, in <module>
    utils.log_print(printFlag,Aw[startV:startV+deltaV, startV:startV+deltaV])
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "tensor([[0.0000, 0.0000, 0.0000, 0.0100, 0.0000, 0.0000, 0.0100],\n        [0.0000, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000, 0.0100],\n        [0.0000, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000],\n        [0.0100, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000],\n        [0.0000, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000],\n        [0.0000, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000],\n        [0.0100, 0.0100, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000]],\n       device='cuda:0')"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 180, in <module>
    utils.log_print(printFlag,"\n\n\n.........................initialize..........................")
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: '\n\n\n.........................initialize..........................'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 181, in <module>
    utils.log_print(printFlag,"FileName:", fileName)
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: 'FileName: AA/AA_GEO_identical_10_4_0_8_50_10.0_140000'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 182, in <module>
    utils.log_print(printFlag,"Strains:", paras.strains)
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: 'Strains: 4'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 183, in <module>
    utils.log_print(printFlag,"Using", paras.randomGraph,"random graph model")
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: 'Using GEO random graph model'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 184, in <module>
    utils.log_print(printFlag,"Num of links:", G.number_of_edges())
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: 'Num of links: 400'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 185, in <module>
    utils.log_print(printFlag,"Reverse ZERO:", torch.sum(torch.abs(Zmat2-Aw)))
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "Reverse ZERO: tensor(4.5821e-07, device='cuda:0')"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 186, in <module>
    utils.log_print(printFlag,"Num links weight bigger than 0.1:", torch.sum(Zmat>0.1)-paras.n)
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "Num links weight bigger than 0.1: tensor(0, device='cuda:0')"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 187, in <module>
    utils.log_print(printFlag,"Num links weight bigger than 0.05:", torch.sum(Zmat>0.05)-paras.n)
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "Num links weight bigger than 0.05: tensor(0, device='cuda:0')"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 188, in <module>
    utils.log_print(printFlag,"Num links weight bigger than 0.01:", torch.sum(Zmat>0.01)-paras.n)
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "Num links weight bigger than 0.01: tensor(347, device='cuda:0')"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 189, in <module>
    utils.log_print(printFlag,".............................................................")
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: '.............................................................'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 190, in <module>
    utils.log_print(printFlag,losses[-1]/timeHorizon*100)#
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: '2.420081727905199e-05'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 204, in <module>
    utils.log_print(printFlag,paras.taus[0: paras.strains])
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: '[7.577506790998881, 7.548740489410478, 7.50449092197787, 7.532814664756424]'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 205, in <module>
    utils.log_print(printFlag,paras.R0s[0: paras.strains])
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: '[8.799698037717521, 8.263759972625767, 8.495236793364116, 8.22348679488616]'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 206, in <module>
    utils.log_print(printFlag,myEpi.taus[0])
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "tensor([7.5833, 7.5373, 7.4991, 7.5379], device='cuda:0',\n       grad_fn=<SelectBackward0>)"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 207, in <module>
    utils.log_print(printFlag,(myEpi.taus*myEpi.R0dTaus)[0])
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "tensor([8.7992, 8.2492, 8.4844, 8.2252], device='cuda:0',\n       grad_fn=<SelectBackward0>)"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 213, in <module>
    utils.log_print(printFlag,"err1:", torch.sqrt((PreA-Aw)**2).sum())
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "err1: tensor(2.5282, device='cuda:0', grad_fn=<SumBackward0>)"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 214, in <module>
    utils.log_print(printFlag,"err2:", torch.abs(PreAse-Awse).sum())
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "err2: tensor(22., device='cuda:0')"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 215, in <module>
    utils.log_print(printFlag,"cosine similarity:", evaluate.cosine_similarity(Aw, PreA))
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "cosine similarity: tensor(0.9451, device='cuda:0', grad_fn=<DivBackward0>)"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 216, in <module>
    utils.log_print(printFlag,"cosine similarity 2:", evaluate.cosine_similarity(Awse, PreAse))
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "cosine similarity 2: tensor(0.9871, device='cuda:0')"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 217, in <module>
    utils.log_print(printFlag,"spectral_analysis:", evaluate.spectral_analysis(Aw, PreA))
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "spectral_analysis: tensor(0.9432, device='cuda:0', grad_fn=<DivBackward0>)"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 219, in <module>
    utils.log_print(printFlag,"edge_correctness:", evaluate.edge_correctness(Awse, PreAse))
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "edge_correctness: tensor(0.9871, device='cuda:0')"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 220, in <module>
    utils.log_print(printFlag,"jaccard_index:", evaluate.jaccard_index(Awse, PreAse))
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "jaccard_index: tensor(0.9744, device='cuda:0')"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 221, in <module>
    utils.log_print(printFlag,torch.var(myEpi.taus, dim= 0))
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "tensor([7.6931e-10, 3.3105e-10, 7.2977e-10, 6.5703e-10], device='cuda:0',\n       grad_fn=<VarBackward0>)"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 222, in <module>
    utils.log_print(printFlag,torch.var(myEpi.R0dTaus, dim= 0))
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "tensor([5.4694e-07, 3.2501e-07, 6.3524e-07, 3.9802e-07], device='cuda:0',\n       grad_fn=<VarBackward0>)"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 223, in <module>
    utils.log_print(printFlag,torch.sum(predSignal[29, :, 0:-1])/paras.strains)
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "tensor(0.9997, device='cuda:0', grad_fn=<DivBackward0>)"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 224, in <module>
    utils.log_print(printFlag,torch.sum(signal[29, :, 1:])/paras.strains)
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "tensor(0.9996, device='cuda:0')"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 227, in <module>
    utils.log_print(printFlag,PreA[startV:startV+deltaV, startV:startV+deltaV])
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "tensor([[0.0000e+00, 3.8737e-03, 1.0167e-02, 1.5239e-02, 1.4107e-05, 1.5724e-03,\n         8.2139e-03],\n        [3.2886e-03, 0.0000e+00, 1.0610e-03, 1.3886e-03, 2.3146e-03, 9.9478e-04,\n         8.1625e-03],\n        [1.7065e-02, 1.5800e-03, 0.0000e+00, 9.2756e-03, 2.4374e-05, 6.0799e-06,\n         4.7049e-03],\n        [1.8409e-02, 1.2375e-03, 7.3196e-03, 0.0000e+00, 4.7034e-06, 1.1993e-03,\n         6.7490e-03],\n        [2.5657e-06, 2.3589e-05, 1.6522e-05, 1.9476e-06, 0.0000e+00, 6.9651e-03,\n         1.5461e-06],\n        [1.9371e-03, 9.7558e-05, 1.0942e-05, 1.8420e-03, 4.0869e-03, 0.0000e+00,\n         7.8204e-05],\n        [1.4305e-02, 9.6652e-03, 9.9717e-03, 8.8739e-03, 7.7743e-05, 1.2744e-03,\n         0.0000e+00]], device='cuda:0', grad_fn=<SliceBackward0>)"
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1087, in emit
    self.flush()
  File "/home/lx/.conda/envs/tf/lib/python3.9/logging/__init__.py", line 1067, in flush
    self.stream.flush()
OSError: [Errno 116] Stale file handle
Call stack:
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/run.py", line 228, in <module>
    utils.log_print(printFlag,Aw[startV:startV+deltaV, startV:startV+deltaV])
  File "/sise/home/lx/code/12_1_topology_learning/Topology_Learning_Epis/modules/utils.py", line 42, in log_print
    logging.info(' '.join(map(str, args)))
Message: "tensor([[0.0000, 0.0000, 0.0100, 0.0100, 0.0000, 0.0000, 0.0100],\n        [0.0000, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000, 0.0100],\n        [0.0100, 0.0000, 0.0000, 0.0100, 0.0000, 0.0000, 0.0100],\n        [0.0100, 0.0000, 0.0100, 0.0000, 0.0000, 0.0000, 0.0100],\n        [0.0000, 0.0000, 0.0000, 0.0000, 0.0000, 0.0100, 0.0000],\n        [0.0000, 0.0000, 0.0000, 0.0000, 0.0100, 0.0000, 0.0000],\n        [0.0100, 0.0100, 0.0100, 0.0100, 0.0000, 0.0000, 0.0000]],\n       device='cuda:0')"
Arguments: ()
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 11 --strains 4  --dense 4 --CMDprogress_3_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA            --seed 10 --strains 4  --n 100 --CMDprogress_4_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 11 --strains 4  --identicalf 0.01 --CMDprogress_5_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA            --seed 14 --strains 4  --n 50 --CMDprogress_6_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 11 --strains 4  --intense 1 --CMDprogress_7_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 13 --strains 4  --dense 8 --CMDprogress_8_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS            --seed 14 --strains 4  --n 150 --CMDprogress_9_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 13 --strains 4  --identicalf 0.02 --CMDprogress_10_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 10 --strains 4  --intense 2 --CMDprogress_11_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 10 --strains 4  --intense 1 --CMDprogress_12_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 13 --strains 4  --identicalf 0.005 --CMDprogress_13_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 10 --strains 4  --dense 6 --CMDprogress_14_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 12 --strains 4  --identicalf 0.005 --CMDprogress_15_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 14 --strains 4  --intense 2 --CMDprogress_16_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER            --seed 12 --strains 4  --n 150 --CMDprogress_17_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO            --seed 11 --strains 4  --n 50 --CMDprogress_18_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 11 --strains 4  --dense 6 --CMDprogress_19_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 14 --strains 4  --identicalf 0.02 --CMDprogress_20_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER            --seed 10 --strains 4  --n 100 --CMDprogress_21_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 14 --strains 4  --dense 6 --CMDprogress_22_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA            --seed 13 --strains 4  --n 150 --CMDprogress_23_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 12 --strains 4  --intense 2 --CMDprogress_24_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 11 --strains 4  --dense 6 --CMDprogress_25_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 13 --strains 4  --dense 6 --CMDprogress_26_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 14 --strains 4  --identicalf 0.01 --CMDprogress_27_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA            --seed 12 --strains 4  --n 50 --CMDprogress_28_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 11 --strains 4  --intense 1 --CMDprogress_29_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 11 --strains 4  --identicalf 0.02 --CMDprogress_30_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 12 --strains 4  --dense 8 --CMDprogress_31_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 11 --strains 4  --dense 8 --CMDprogress_32_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 12 --strains 4  --intense 1 --CMDprogress_33_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 13 --strains 4  --identicalf 0.005 --CMDprogress_34_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 14 --strains 4  --identicalf 0.02 --CMDprogress_35_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 12 --strains 4  --dense 8 --CMDprogress_36_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO            --seed 14 --strains 4  --n 150 --CMDprogress_37_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 14 --strains 4  --dense 4 --CMDprogress_38_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 11 --strains 4  --dense 8 --CMDprogress_39_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 12 --strains 4  --intense 1 --CMDprogress_40_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 11 --strains 4  --identicalf 0.02 --CMDprogress_41_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 10 --strains 4  --dense 8 --CMDprogress_42_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 11 --strains 4  --identicalf 0.005 --CMDprogress_43_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 13 --strains 4  --intense 2 --CMDprogress_44_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 10 --strains 4  --dense 4 --CMDprogress_45_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 11 --strains 4  --dense 8 --CMDprogress_46_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 12 --strains 4  --intense 2 --CMDprogress_47_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 10 --strains 4  --dense 6 --CMDprogress_48_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 12 --strains 4  --intense 1 --CMDprogress_49_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 12 --strains 4  --identicalf 0.02 --CMDprogress_50_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 13 --strains 4  --identicalf 0.02 --CMDprogress_51_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 14 --strains 4  --identicalf 0.01 --CMDprogress_52_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER            --seed 13 --strains 4  --n 150 --CMDprogress_53_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 11 --strains 4  --intense 1 --CMDprogress_54_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO            --seed 12 --strains 4  --n 100 --CMDprogress_55_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 10 --strains 4  --intense 2 --CMDprogress_56_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 12 --strains 4  --identicalf 0.005 --CMDprogress_57_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 12 --strains 4  --intense 2 --CMDprogress_58_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 13 --strains 4  --dense 8 --CMDprogress_59_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 14 --strains 4  --intense 0 --CMDprogress_60_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 13 --strains 4  --identicalf 0.01 --CMDprogress_61_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 12 --strains 4  --identicalf 0.01 --CMDprogress_62_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 13 --strains 4  --intense 0 --CMDprogress_63_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 12 --strains 4  --dense 4 --CMDprogress_64_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 10 --strains 4  --identicalf 0.005 --CMDprogress_65_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 12 --strains 4  --dense 4 --CMDprogress_66_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 11 --strains 4  --identicalf 0.01 --CMDprogress_67_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 14 --strains 4  --identicalf 0.005 --CMDprogress_68_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 11 --strains 4  --dense 4 --CMDprogress_69_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER            --seed 12 --strains 4  --n 50 --CMDprogress_70_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 10 --strains 4  --intense 1 --CMDprogress_71_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 14 --strains 4  --intense 2 --CMDprogress_72_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 13 --strains 4  --intense 0 --CMDprogress_73_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 14 --strains 4  --intense 0 --CMDprogress_74_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 12 --strains 4  --identicalf 0.02 --CMDprogress_75_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 14 --strains 4  --identicalf 0.01 --CMDprogress_76_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 12 --strains 4  --dense 4 --CMDprogress_77_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 12 --strains 4  --identicalf 0.02 --CMDprogress_78_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 11 --strains 4  --identicalf 0.01 --CMDprogress_79_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER            --seed 11 --strains 4  --n 50 --CMDprogress_80_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA            --seed 10 --strains 4  --n 150 --CMDprogress_81_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 10 --strains 4  --intense 1 --CMDprogress_82_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO            --seed 13 --strains 4  --n 150 --CMDprogress_83_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 10 --strains 4  --intense 1 --CMDprogress_84_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 10 --strains 4  --dense 6 --CMDprogress_85_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS            --seed 10 --strains 4  --n 50 --CMDprogress_86_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 10 --strains 4  --intense 0 --CMDprogress_87_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA            --seed 14 --strains 4  --n 150 --CMDprogress_88_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 11 --strains 4  --dense 4 --CMDprogress_89_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 12 --strains 4  --dense 8 --CMDprogress_90_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 11 --strains 4  --dense 6 --CMDprogress_91_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 14 --strains 4  --intense 1 --CMDprogress_92_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 11 --strains 4  --identicalf 0.005 --CMDprogress_93_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 14 --strains 4  --identicalf 0.005 --CMDprogress_94_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER            --seed 13 --strains 4  --n 100 --CMDprogress_95_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 12 --strains 4  --dense 8 --CMDprogress_96_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER            --seed 10 --strains 4  --n 50 --CMDprogress_97_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER            --seed 11 --strains 4  --n 150 --CMDprogress_98_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 11 --strains 4  --intense 0 --CMDprogress_99_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 14 --strains 4  --dense 6 --CMDprogress_100_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS            --seed 11 --strains 4  --n 50 --CMDprogress_101_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO            --seed 10 --strains 4  --n 150 --CMDprogress_102_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA            --seed 11 --strains 4  --n 100 --CMDprogress_103_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 14 --strains 4  --identicalf 0.005 --CMDprogress_104_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO            --seed 11 --strains 4  --n 100 --CMDprogress_105_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA            --seed 12 --strains 4  --n 100 --CMDprogress_106_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 13 --strains 4  --identicalf 0.005 --CMDprogress_107_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA            --seed 12 --strains 4  --n 150 --CMDprogress_108_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO            --seed 10 --strains 4  --n 50 --CMDprogress_109_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 14 --strains 4  --dense 8 --CMDprogress_110_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 10 --strains 4  --dense 4 --CMDprogress_111_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO            --seed 14 --strains 4  --n 50 --CMDprogress_112_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 13 --strains 4  --dense 4 --CMDprogress_113_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER            --seed 14 --strains 4  --n 50 --CMDprogress_114_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 10 --strains 4  --identicalf 0.02 --CMDprogress_115_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 10 --strains 4  --dense 4 --CMDprogress_116_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS            --seed 13 --strains 4  --n 50 --CMDprogress_117_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 13 --strains 4  --dense 6 --CMDprogress_118_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 13 --strains 4  --dense 8 --CMDprogress_119_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 14 --strains 4  --intense 2 --CMDprogress_120_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 11 --strains 4  --dense 8 --CMDprogress_121_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 10 --strains 4  --identicalf 0.005 --CMDprogress_122_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 12 --strains 4  --intense 2 --CMDprogress_123_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 14 --strains 4  --identicalf 0.005 --CMDprogress_124_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 12 --strains 4  --identicalf 0.01 --CMDprogress_125_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 10 --strains 4  --identicalf 0.02 --CMDprogress_126_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 14 --strains 4  --dense 8 --CMDprogress_127_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 11 --strains 4  --identicalf 0.01 --CMDprogress_128_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 10 --strains 4  --dense 6 --CMDprogress_129_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 14 --strains 4  --intense 1 --CMDprogress_130_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 12 --strains 4  --identicalf 0.01 --CMDprogress_131_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 13 --strains 4  --identicalf 0.02 --CMDprogress_132_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 12 --strains 4  --intense 0 --CMDprogress_133_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 11 --strains 4  --intense 2 --CMDprogress_134_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 13 --strains 4  --intense 0 --CMDprogress_135_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS            --seed 12 --strains 4  --n 100 --CMDprogress_136_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 14 --strains 4  --intense 1 --CMDprogress_137_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 11 --strains 4  --intense 2 --CMDprogress_138_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 12 --strains 4  --intense 0 --CMDprogress_139_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 13 --strains 4  --dense 6 --CMDprogress_140_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER            --seed 13 --strains 4  --n 50 --CMDprogress_141_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 13 --strains 4  --intense 1 --CMDprogress_142_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 11 --strains 4  --intense 2 --CMDprogress_143_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 13 --strains 4  --dense 4 --CMDprogress_144_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 11 --strains 4  --identicalf 0.005 --CMDprogress_145_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 12 --strains 4  --identicalf 0.01 --CMDprogress_146_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 13 --strains 4  --intense 0 --CMDprogress_147_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 14 --strains 4  --identicalf 0.02 --CMDprogress_148_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 13 --strains 4  --dense 4 --CMDprogress_149_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER            --seed 14 --strains 4  --n 150 --CMDprogress_150_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA            --seed 14 --strains 4  --n 100 --CMDprogress_151_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS            --seed 10 --strains 4  --n 150 --CMDprogress_152_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO            --seed 10 --strains 4  --n 100 --CMDprogress_153_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 13 --strains 4  --intense 2 --CMDprogress_154_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS            --seed 13 --strains 4  --n 100 --CMDprogress_155_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS            --seed 10 --strains 4  --n 100 --CMDprogress_156_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER            --seed 11 --strains 4  --n 100 --CMDprogress_157_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 10 --strains 4  --intense 2 --CMDprogress_158_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 10 --strains 4  --dense 8 --CMDprogress_159_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 10 --strains 4  --dense 8 --CMDprogress_160_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 14 --strains 4  --dense 6 --CMDprogress_161_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER            --seed 10 --strains 4  --n 150 --CMDprogress_162_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER            --seed 12 --strains 4  --n 100 --CMDprogress_163_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA            --seed 13 --strains 4  --n 100 --CMDprogress_164_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO            --seed 13 --strains 4  --n 50 --CMDprogress_165_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 14 --strains 4  --intense 2 --CMDprogress_166_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 11 --strains 4  --identicalf 0.005 --CMDprogress_167_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 13 --strains 4  --identicalf 0.01 --CMDprogress_168_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 11 --strains 4  --intense 0 --CMDprogress_169_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 12 --strains 4  --intense 0 --CMDprogress_170_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 12 --strains 4  --intense 0 --CMDprogress_171_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 10 --strains 4  --intense 0 --CMDprogress_172_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 14 --strains 4  --dense 8 --CMDprogress_173_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER            --seed 14 --strains 4  --n 100 --CMDprogress_174_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 10 --strains 4  --identicalf 0.005 --CMDprogress_175_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 14 --strains 4  --identicalf 0.02 --CMDprogress_176_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 14 --strains 4  --intense 0 --CMDprogress_177_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 14 --strains 4  --identicalf 0.01 --CMDprogress_178_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 11 --strains 4  --dense 6 --CMDprogress_179_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 11 --strains 4  --dense 4 --CMDprogress_180_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 12 --strains 4  --dense 6 --CMDprogress_181_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 12 --strains 4  --intense 1 --CMDprogress_182_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 13 --strains 4  --intense 1 --CMDprogress_183_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 10 --strains 4  --intense 2 --CMDprogress_184_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO            --seed 11 --strains 4  --n 150 --CMDprogress_185_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 10 --strains 4  --identicalf 0.005 --CMDprogress_186_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA            --seed 11 --strains 4  --n 150 --CMDprogress_187_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 14 --strains 4  --dense 4 --CMDprogress_188_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 11 --strains 4  --intense 1 --CMDprogress_189_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO            --seed 12 --strains 4  --n 150 --CMDprogress_190_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 13 --strains 4  --identicalf 0.02 --CMDprogress_191_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 13 --strains 4  --dense 8 --CMDprogress_192_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 10 --strains 4  --identicalf 0.02 --CMDprogress_193_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO            --seed 12 --strains 4  --n 50 --CMDprogress_194_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 10 --strains 4  --dense 8 --CMDprogress_195_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 13 --strains 4  --intense 2 --CMDprogress_196_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS            --seed 14 --strains 4  --n 50 --CMDprogress_197_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 10 --strains 4  --identicalf 0.02 --CMDprogress_198_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 14 --strains 4  --dense 4 --CMDprogress_199_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 12 --strains 4  --dense 6 --CMDprogress_200_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS            --seed 11 --strains 4  --n 100 --CMDprogress_201_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 11 --strains 4  --identicalf 0.02 --CMDprogress_202_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 10 --strains 4  --intense 0 --CMDprogress_203_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 12 --strains 4  --identicalf 0.02 --CMDprogress_204_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 13 --strains 4  --identicalf 0.01 --CMDprogress_205_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 11 --strains 4  --intense 0 --CMDprogress_206_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS            --seed 13 --strains 4  --n 150 --CMDprogress_207_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 13 --strains 4  --intense 2 --CMDprogress_208_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 13 --strains 4  --dense 4 --CMDprogress_209_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 12 --strains 4  --dense 6 --CMDprogress_210_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 14 --strains 4  --dense 6 --CMDprogress_211_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS            --seed 14 --strains 4  --n 100 --CMDprogress_212_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 14 --strains 4  --intense 0 --CMDprogress_213_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 13 --strains 4  --intense 1 --CMDprogress_214_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA            --seed 13 --strains 4  --n 50 --CMDprogress_215_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 10 --strains 4  --identicalf 0.01 --CMDprogress_216_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 12 --strains 4  --dense 4 --CMDprogress_217_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS            --seed 12 --strains 4  --n 50 --CMDprogress_218_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 13 --strains 4  --intense 1 --CMDprogress_219_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS            --seed 12 --strains 4  --n 150 --CMDprogress_220_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 11 --strains 4  --identicalf 0.02 --CMDprogress_221_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 12 --strains 4  --identicalf 0.005 --CMDprogress_222_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 11 --strains 4  --intense 0 --CMDprogress_223_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA            --seed 10 --strains 4  --n 50 --CMDprogress_224_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 13 --strains 4  --dense 6 --CMDprogress_225_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 14 --strains 4  --dense 4 --CMDprogress_226_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 10 --strains 4  --identicalf 0.01 --CMDprogress_227_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO            --seed 14 --strains 4  --n 100 --CMDprogress_228_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS              --seed 11 --strains 4  --intense 2 --CMDprogress_229_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 12 --strains 4  --identicalf 0.005 --CMDprogress_230_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph WS            --seed 11 --strains 4  --n 150 --CMDprogress_231_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 12 --strains 4  --dense 6 --CMDprogress_232_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA              --seed 10 --strains 4  --intense 0 --CMDprogress_233_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 14 --strains 4  --intense 1 --CMDprogress_234_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO              --seed 13 --strains 4  --identicalf 0.01 --CMDprogress_235_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph BA            --seed 11 --strains 4  --n 50 --CMDprogress_236_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph GEO            --seed 13 --strains 4  --n 100 --CMDprogress_237_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 13 --strains 4  --identicalf 0.005 --CMDprogress_238_239
python run.py --epoches 140000 --modelLoad AA --weightModel identical --randomGraph ER              --seed 10 --strains 4  --identicalf 0.01 --CMDprogress_239_239
